README.md

```diff
@@ -0,0 +1,77 @@
+# JavaScript Capstone Restuarant APP using API
+
+> This is a Restuarant App Developed in Vanilla JavaScript. The main objective for this project is to learn about basic JavaScript concepts, functions, arrays and objects DOM manipulation and connect to API's.
+
+### Screenshots
+
+|Main Project page
+
+|![Fastantistic Restaurant](https://user-images.githubusercontent.com/30318155/129350424-7d1d5391-4d25-44d0-94a0-b08659b2d95a.png)|
+
+## Built With
+
+- HTML5
+- CSS3
+- Bootstrap
+- Font Awesome
+- JavaScript-
+- MDB API
+- Involvement API
+- Jest Test
+
+## Our Project
+
+This project is found on the following Repo [Fantastic Restaurant](https://github.com/tnyandoro/JavaScript-Capstone/)
+
+## Live Demo
+
+You can view the live demo of the project [Here!](https://tnyandoro.github.io/JavaScript-Capstone/)
+
```

Comment:
- [ ] The deployed version of the project contains a text at the page bottom that doesn't belong there, kindly remove it.

<img src='https://user-images.githubusercontent.com/33323077/129366590-05410c8e-f98b-4752-ad49-e09386b2a239.png' height='250px'>

Reply:
Error resolved